### PR TITLE
Added delete_external_event method

### PR DESCRIPTION
**Pull request by stephenbinns**

### CODE REVIEW

File: lib/cronofy/client.rb [hunk 1]
```diff
@@ -270,6 +270,30 @@ def delete_event(calendar_id, event_id)
       nil
     end
 
+    # Public: Deletes an external event from the specified calendar
+    #
+    # calendar_id - The String Cronofy ID for the calendar to delete the event
+    #               from.
+    # event_uid   - The unique ID of the event to delete
+    #               (note: this is NOT an ID generated by Cronofy).
```
Comment (gshutler) [inline lib/cronofy/client.rb]: False.

Comment (stephenbinns) [inline lib/cronofy/client.rb]: Which bit?

File: lib/cronofy/client.rb [hunk 2]
```diff
@@ -270,6 +270,30 @@ def delete_event(calendar_id, event_id)
...
+    #
+    # Returns nothing.
+    #
+    # Raises Cronofy::CredentialsMissingError if no credentials available.
+    # Raises Cronofy::AuthenticationFailureError if the access token is no
+    # longer valid.
+    # Raises Cronofy::AuthorizationFailureError if the access token does not
+    # include the required scope.
```
Comment (gshutler) [inline lib/cronofy/client.rb]: May also be permissions, at least worth mentioning I think.

File: lib/cronofy/client.rb (hunk 1 above)
Comment (gshutler) [inline lib/cronofy/client.rb]: The note, the `event_uid` is our ID for the event.

**Comment by gshutler** [comment]
👍

**Pull request merged**